ibis/backends/tests/test_temporal.py

```diff
@@ -1260,12 +1267,18 @@ def test_integer_to_timestamp(backend, con, unit):
 @pytest.mark.notimpl(["exasol"], raises=com.OperationNotDefinedError)
 def test_string_to_timestamp(alltypes, fmt):
     table = alltypes
-    result = table.mutate(date=table.date_string_col.to_timestamp(fmt)).execute()
+    result_as = table.mutate(date=table.date_string_col.as_timestamp(fmt)).execute()
+
+    with pytest.warns(FutureWarning, match="v10.0"):
```

Comment:
I had this here, to check that we were still doing the right thing with `to_*` but I think this is causing the pyspark ones to fail. 

From Phillip ins a similar situation: "a few of the driver libraries are really bad about closing sockets (or more generally cleaning up the things that create sockets)"

Maybe we don't test this...